resources/views/store/partials/add_child_form.blade.php

```diff
@@ -88,17 +88,22 @@ function () {
                 var innerTextDate = dateObj.format("MMM YYYY");
                 var valueDate = dateObj.format("YYYY-MM");
 
-                // Make some age display values
-                var displayMonths = moment().diff(valueDate, 'months') % 12;
-                var displayYears = moment().diff(valueDate, 'years');
+                // Make some age display values based on if it's the future.
+                var ageString = (moment().diff(valueDate, 'days') > 0)
+                    ? '<td class="age-col">' +
+                        moment().diff(valueDate, 'years') + ' yr, ' +
+                        moment().diff(valueDate, 'months') % 12  +
+                        ' mo</td>'
+                    : '<td class="age-col">P</td>'
```

Comment:
What's the P?

Reply:
It indicates pregnancy.

Reply:
Having negative dates to represent unborn kids looked "wierd" to CC users, and hard to pick out. We went with "P" as they understood that better.